File: ide/eclipse/L0202Types/src/cn/eoe/types/Man.java
package cn.eoe.types;

public class Man extends People {

	public Man(int age) {
		super(age);
	}
	
	public void fuckWoman(){
		
	}
	
	
	@Override
	public int getAge() {
		System.out.println("Man get age");
		return super.getAge();
	}

}

File: ide/xcode/L0202HelloCPP/L0202HelloCPP/Man.cpp
//
//  Man.cpp
//  L0202HelloCPP
//
//  Created by plter on 13-10-12.
//  Copyright (c) 2013年 plter. All rights reserved.
//

#include "Man.h"

namespace eoe {
    
    Man::Man(int age,string name):People(age,name,"男"){
    }
    
    void Man::sayHi(){
        cout<<"Man say hi\n";
    }
    
    void Man::eat(){
        cout<<"Man eat \n";
    }
    
    void Man::fuck(eoe::Woman *wm){
        
    }
}